brand/models/commentary.py

```diff
@@ -177,6 +180,15 @@ def compute_inherited_rating(self, inheritance_set=None, throw_error=False):
         blank=True,
     )
 
+    feature_refresh_date = models.DateTimeField(null=True, blank=True)
+    feature_json = models.JSONField(null=True, blank=True)
```

Comment:
Add `default=dict` here and then delete the old migration and `python manage.py makemigrations`. There's nothing wrong with what's already here, but I've noticed that saving an empty dictionary can otherwise prove problematic.